### ISSUE TITLE
Add some sort of timer filter for the dev-spotter function

### ISSUE THREAD

**Issue by kokarn** [post-tracker/twitch-lurker#4]
Right now it will log a spotting every *number of streams* seconds.
Maybe set a limit to once every hour?

**Comment by injectJon** [comment]
We could leave it as is now, and just monitor changes to their state.
If their state changes ( e.g. they switch to a new stream ) we should always output that, but if their state hasn't changed after an hour, we output their current state again?

**Comment by kokarn** [comment]
Yeah, that's what I'm proposing at least. Because I assume we wanna see if a person leaves a stream for say 4 hours and then comes back. But that might not be true, not sure
